DMTN-118.tex

```diff
@@ -46,6 +50,683 @@
+In both cases forced photometry is performed both on the difference image and the unsubtracted processed visit image, so the discussion in \S \ref{sec:totFlux} also applies to forced photometry.
+
+\textit{Should we use forced photometry measurements (either on difference images or PVIs) to compute timeseries features in AP?}
+If we only compute features when \DIASources are detected, variable star photometry will be highly biased, as no \DIASources will be present when the variable star flux is near the template value.
+However, PSF forced photometry fluxes are sensitive to centroiding errors, whether due to bad image subtractions or errors in modeling proper motion.
+Moreover, for objects that have faded significantly many forced photometry measurements will simply record zero flux with background noise, which could dilute the signal for some features.
+\citet{Sanchez-Saez:21:AlertClassification} achieved reasonable performance in classifying variable stars from ZTF alerts despite the absence of forced photometry measurements.
+
+There is also a subtlety in interpreting the alert contents.
+Alerts are only triggered by \DIASource \textit{detections}, but if we use forced photometry lightcurves for timeseries features the latest measurement will not be identical to the triggering \DIASource.
+Also, \DIAObjects would require updating each time a new forced photometery measurement was made, which is not in the current processing plan.
+
+If we did decide that we wanted to update \DIAObject features with forced photometry results (and no triggering \DIASource) we could do so in a more leisurely way---these would be subject to L1PublicT rather than OTT1 requirements.
+
+In DRP uniform forced photometry lightcurves will be available for all \Objects and \DIAObjects, so it will make sense to use these to compute the timeseries features.
+
+\subsection{Times}
+
+Period-finding and other computations for short-timescale events can be biased by light-travel time across the solar system.
+Accordingly our feature computation should should use barycentered times.  
+Pipelines will natively report TAI times.
+\textit{Should we also store the barycentered times in the \DIASource record?}
+The transformation is straightforward to compute; however, if the barycentered time is omitted users will have to recognize that they cannot reproduce our features without first computing the barycentered times.
+
+\subsection{Special Programs}
+
+Observations of Special Programs might benefit from different timeseries features to capitalize on the potential changes in image characteristics, time sampling, and total number of observations.
+\textit{Would special programs benefit from unique timeseries features, and could this be accommodated technically?}
+As discussed in \citeds{DMTN-065}, the baseline is to process Special Programs data identically when it is possible for DM to do so. 
```

Comment:
Special prgrams data will be processed using the same algorithms but those algorithms can be reconfigured, so I wouldn't say that the Special Programs data will be processed identically.